affiliated/registry.json

```
@@ -1015,9 +1015,9 @@
         },
         {
             "name": "sbpy",
-            "maintainer": "Michael Mommert, Michael S. P. Kelley, Miguel de Val-Borro, and Jian-Yang Li <sbpy.dev@gmail.com>",
+            "maintainer": "Michael S. P. Kelley <msk@astro.umd.edu>, Henry Hsieh, and Jian-Yang Li",
```

I think email has to be at the end. The rendered page does not have your collaborators listed for some reason.
```suggestion
            "maintainer": "Michael S. P. Kelley, Henry Hsieh, and Jian-Yang Li <msk@astro.umd.edu>",
```

---

Thanks, I think it is fixed now.